pom.xml

```diff
@@ -22,7 +22,7 @@
   <properties>
     <revision>1.9</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.164.1</jenkins.version>
```

Comment:
integrations/pom.xml should have the jenkins.version removed

Reply:
Plenty of deprecated functions to go through as well :smile: